docs/references/chrome-extension/add-react-router.mdx

```diff
@@ -17,7 +17,7 @@ description: Learn how to add React Router to your React application using their
       icon: "clerk"
     },
     {
-      title: "Add authentication and user management to your Chrome Extension with Clerk",
+      title: "Add auth and user management to your Chrome Extension with Clerk",
       link: "/docs/quickstarts/chrome-extension",
       icon: "react"
```

Comment:
Should this be the Chrome Extension icon? Or the Clerk icon even? The React one feels weird here.

Reply:
```suggestion
      icon: "chrome-extension"
```

Reply:
Did you request the Design team to make a Chrome Extension icon, and then upload it accordingly to the `clerk/clerk` repo? If not, I can do it for you, just let me know

Reply:
No, I didn't know it needed to be in that repo. I only got the icon for the side bar done.